mkdocs.yml

```
@@ -1,20 +1,95 @@
 site_name: Guidelines and Documentation
 pages:
 - Home: 'index.md'
+- Platform Features:
+  - Structure:
+    - 'Courses': 'features/structure/courses.md'
+    - 'Sections': 'features/structure/sections.md'
+    - 'Items': 'features/structure/items.md'
+  - Item Types:
+    - 'Video': 'features/itemtypes/video.md'
+    - 'Quiz': 'features/itemtypes/quiz.md'
+    - 'Surveys': 'features/itemtypes/surveys.md'
+    - 'Text': 'features/itemtypes/text.md'
+    - 'LTI': 'features/itemtypes/lti.md'
+    - 'Peer Assessment': 'features/itemtypes/peerassessment.md'
+   - Discussion Forum: 'features/discussionforum.md'
+   - Collab spaces: 'features/collabspaces.md'
+   - Gamification: 'features/gamification.md'
```

The indention level is invalid. I assume you want to remove one space per line.

```suggestion
  - Discussion Forum: 'features/discussionforum.md'
  - Collab spaces: 'features/collabspaces.md'
  - Gamification: 'features/gamification.md'
```